client/components/QuestionsAndAnswers/QuestionsAndAnswers.jsx

```diff
@@ -11,21 +11,24 @@ const QuestionsAndAnswers = (props) => {
   const [ numQuestions, setNumQuestions ] = useState(2);
   //Initial questions set is sorted by helpfulness
   const initialQuestions = tempProductQuestions.results.sort((a, b) => b.question_helpfulness - a.question_helpfulness);
-  const [ questions, setQuestions ] = useState(initialQuestions);
+  const [ questions, setQuestions ] = useState([]);
 
-  curProduct.id = 12013;
   //Initial data fetch
   useEffect(() => {
-    axios.get(`proxy/api/fec2/hratx/qa/questions?product_id=${curProduct.id}`)
+    axios.get(`proxy/api/fec2/hratx/qa/questions?product_id=${curProduct.id}&count=50`)
       .then(res => {
         setQuestions(res.data.results);
       });
   }, [curProduct.id]);
 
-  const handleAddQuestion = () => {
-    // console.log('Adding question!');
-    //TODO
-  };
+  const handleAddQuestion = (question) => {
+    axios.post(`proxy/api/fec2/hratx/qa/questions`, question)
+    .then(() => {
+        setQuestions([...questions, { question_id: questions.length + 1, question_body: question.body, question_helpfulness: 0, asker_name: question.name}]);
+    })
```

Comment:
Everything looks good. Only nit pick I have is to maybe separate the key value pairs onto different lines such as right here for readability, but ultimately it is fine either way.